public/app/features/alerting/unified/utils/misc.ts

```diff
@@ -38,7 +38,8 @@ export const getFiltersFromUrlParams = (queryParams: UrlQueryMap): FilterState =
   const alertState = queryParams['alertState'] === undefined ? undefined : String(queryParams['alertState']);
   const dataSource = queryParams['dataSource'] === undefined ? undefined : String(queryParams['dataSource']);
   const groupBy = queryParams['groupBy'] === undefined ? undefined : String(queryParams['groupBy']).split(',');
-  return { queryString, alertState, dataSource, groupBy };
+  const silenceState = queryParams['silenceState'] === undefined ? undefined : String(queryParams['silenceState']);
+  return { queryString, alertState, dataSource, groupBy, silenceState };
```

Comment:
maybe have a separate function for silences? It's confusing to use the same for different kinds of filters